### ISSUE TITLE
Running DOC Tests only if NumPy >= 1.14 

### ISSUE THREAD

**Issue by KamalakerDadi** [nilearn/nilearn#1593]
In the long run we should plan to update all the doc tests to run only if NumPy version >= 1.14. NumPy had made more consistent the formatting of output arrays. 
For example:

```python
Before NumPy 1.14:
    array([[ 0.,  1.],
           [ 1.,  0.]])
From NumPy 1.14:
    array([[0., 1.],
           [1., 0.]])
```
This new enhancement will more likely breaks the doc tests. A temporary work around to switch back to NumPy 1.13 formatting can be seen in this PR #1592

**Comment by Remi-Gau** [comment]
I think this can be closed given we only use numpy>=1.19.0

**Comment by bthirion** [comment]
Indeed.
